src/decompiler.cc

```diff
@@ -58,6 +56,14 @@ struct Decompiler {
     Value& operator=(const Value& rhs) = delete;
   };
 
+  std::string to_string(double d) {
+    auto s = std::to_string(d);
+    // Remove redundant trailing '0's that to_string produces.
+    while (s.size() > 2 && s.back() == '0' && s[s.size() - 2] != '.')
+      s.pop_back();
```

Comment:
I think wabt style requires curlies here.

Reply:
Looks like WABT uses `Chromium` style, which is an extension of `Google` style, which allows these curly-less statements.